src/sentry/api/endpoints/integrations/sentry_apps/index.py

```diff
@@ -88,9 +90,9 @@ def post(self, request: Request, organization) -> Response:
             "schema": request.json_body.get("schema", {}),
             "overview": request.json_body.get("overview"),
             "allowedOrigins": request.json_body.get("allowedOrigins", []),
-            "popularity": request.json_body.get("popularity")
-            if is_active_superuser(request)
-            else None,
+            "popularity": (
+                request.json_body.get("popularity") if is_active_superuser(request) else None
```

Comment:
should this use `elevated_user` as well?

Reply:
Staff won't have access to the POST so it should be fine to leave as just superuser